include/rcutils/filesystem.h

```diff
@@ -187,7 +187,8 @@ rcutils_mkdir(const char * abs_path);
 /// Calculate the size of the specified directory.
 /*
  * Calculates the size of a directory by summarizing the file size of all files.
- * \note This operation is not recursive.
+ * \note This operation is recursive.
```

Comment:
It seems that the recursive operation does not contain any subdirectory symbolic links. Do you think we need to update the note with this kind of message?

Reply:
Nice catch! i do not really see the use cases for plugin creates the symlink. but it should be clearly defined and described in the header.